ddp-workspace/projects/toolkit-prion/src/lib/components/activity/prionActivity.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { ActivityComponent, ToolkitConfigurationService, WorkflowBuilderService } from 'toolkit';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'prion-activity',
+  template: `
+    <ng-container>
+        <prion-header>
+        </prion-header>
+        <div class="ContainerSurvey-top row">
+        <ddp-activity [studyGuid]="studyGuid"
+                      [activityGuid]="id"
+                      (submit)="navigate($event)"
+                      class="ContainerSurvey col-lg-8 col-lg-offset-2 col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 col-xs-12">
+        </ddp-activity>
+        </div>
+    </ng-container>`
+})
+export class PrionActivityComponent extends ActivityComponent implements OnInit {
```

Comment:
OnInit is not implmented:
```suggestion
export class PrionActivityComponent extends ActivityComponent  {
```